### ISSUE TITLE
Error in Parsing PoolingND

### ISSUE THREAD

**Issue by User-Ali** [qxcv/caffe2keras#4]
I don't know much about Caffe. But, when I google to resolve this issue, I learned that there is no poolingnd in  Caffe but some people were referring cudNN for it. 

Now, can you please suggest how can I resolve this issue? PS : I don't have GPU so, no cudNN either. 

Here is the layer:
`layer {   name: "pool1"   type: "PoolingND"   bottom: "conv1H"   top: "pool1"   poolingnd_param {     pool: MAX     kernel_h: 2     kernel_w: 2     kernel_d: 2     stride: 2   } }`

**Comment by qxcv** [comment]
I don't think that error has anything to do with CuDNN. It looks like your model has an unusual layer from a third-party Caffe fork (maybe Caffe3D?). This converter only supports the default Caffe layers at the moment; patches are welcome if you'd like to extend it to support whatever fork you're using.

**Comment by User-Ali** [comment]
I wish I could but I don't have relevant knowledge. I could try to work on that if you provide some guidelines. Here is a related [post](https://github.com/faustomilletari/3D-Caffe/issues/4) on PoolingND.

 It's used as a 3D Pooling layer but they have nothing mentioned about which Caffe version they used for it.
